compiler/generator_rust.c

```diff
@@ -1188,14 +1188,23 @@ static void generate(struct generator * g, struct node * p) {
 }
 
 static void generate_start_comment(struct generator * g) {
-
```

Comment:
Sorry, I didn't realize these newlines were intentional. Fixed it by matching `generator_python.c`s newline pattern.